### ISSUE TITLE
Switch to new window upon startup

### ISSUE THREAD

**Issue by jidanni** [viking-gps/viking#213]
I'm using "i3 - an improved dynamic, tiling window manager"

I notice when I do
$ viking file.kmz
I then have to manually click the new tab created to see it. (Same with google-earth-pro.)

Compare that to emacs, chrome, etc. which switch to the new window created, no clicking needed.

But on a Chromebook, starting viking pops up the viking window with no clicking needed too.

**Comment by rnorris** [comment]
I've tried i3 (version 4.22) on my Debian system, and the system installed Viking (as well as the latest dev version) automatically switches to the new tab taking keyboard focus and also displaying whatever file you have opened.

So I don't think this is an issue with Viking.

**Comment by rnorris** [comment]
Note you can't open .kmz files directly on the command line (nor via the menu File->Open).
You have to **Import** them.
That means using the menu File->Import KMZ Map File...

**Comment by jidanni** [comment]
Huh? I open them all the time from the command line.
```
wget https://www.jidanni.org/geo/house_numbering/grids/tw/taichung/daan/ew_ns/ewns.kmz
viking --map 13 ewns.kmz
```
https://www.jidanni.org/geo/house_numbering/grids/tw/taichung/daan/ew_ns/Makefile

**Comment by rnorris** [comment]
Perhaps I should rephrase the "...can't open .kmz files directly on the command line..." - as it is not recommended to open .kmz files this way as they may not be interpreted as one might expect.

[.kmz](https://developers.google.com/kml/documentation/kmzarchives) file support via Import/Export methods was originally added into Viking (v1.7) mainly to support usage with Garmin GPS devices - see [Garmin Custom Maps]( https://www8.garmin.com/manuals/webhelp/basecamppc/EN-US/GUID-E8D71687-12FB-404E-BBFB-7FA5ED7BAE0C.html) - which are ones with specifically Image files - used as a Map(s) - packaged in the compressed file; rather than a generic zip file with KML contents - which is perhaps what the .kmz is supposed to be (according to Google themselves).

When opening a .kmz file via the command-line, Viking (since v1.8) will detect is it a zip file, uncompress it and read any geo-data files found with in it. However in this reading of KML files, Viking will only extract Tracks, Routes and Waypoints and not any other 'advanced' KML attributes. So if your .kmz contains just 'simple' geodata - then it will appear to work fine. 

ATM only via the KMZ Import will these Images under the 'GroundOverlay' XML tags be read in KML files - they are converted into GeoRef Layers in order to align the image to the map area coverage.

HTH

**Comment by jidanni** [comment]
That's fine for me. I only used the simplest of KMZ files. They don't even have folders in them. Just one dimensional.

Then there's the curious minus e option on the man page. Never understood if it also applied to kmzs, or only gpx.

And of course my idea was I could control the whole thing from the command line: selecting the map with the minus m thing. But that's defeated by the default layer.

Well anyway that's what I'm doing. About 100 quick views of different versions of the same kmz an hour.

And I kill the previous Viking before starting the next one with the killall command.

Well I guess I got kind of far off topic.
